data/human/campaign events.txt

```diff
@@ -519,6 +519,80 @@ event "initial deployment 3"
 		fleet "Large Republic" 800
 		fleet "Navy Surveillance" 1200
 
+
+
+mission "Central Outfitters Expansion Trigger"
+	landing
+	invisible
+	to offer
+		has "event: initial deployment 2"
+	on offer
+		event "central outfitters expansion" 45
+
+event "central outfitters expansion"
+	planet "Earth"
+		add outfitter "Central Outfits Expansion"
+	planet "Luna"
+		add outfitter "Central Outfits Expansion"
+	planet "New China"
+		add outfitter "Central Outfits Expansion"
+	planet "New Wales"
+		add outfitter "Central Outfits Expansion"
+	planet "Heartland"
+		add outfitter "Central Outfits Expansion"
+
+mission "Central Outfitters Expansion"
+	outfitter
+	source
+		attributes "near earth" "dirt belt"
+		outfits "Tactical Scanner"
+		outfits "Volcano Afterburner"
+		outfits "Caldera Afterburner"
+		outfits "S3 Thermionic"
+		outfits "Heavy Rocket"
+		outfits "Meteor Missile"
+		outfits "Javelin"
+		outfits "Javelin Pod"
+		outfits "Javelin Mini Pod"
+		outfits "Javelin Storage Crate"
+		outfits "Small Radar Jammer"
+		outfits "Brig"
+		outfits "Liquid Nitrogen Cooler"
```

Comment:
Is the idea here to exclude New Wales and Heartland from getting this mission?

Reply:
No? Though perhaps I can replace the outfits list with the list of planets getting the outfitters.

Reply:
Oh, I've just realised; it's because I'm pretty sure you can't have a source with multiple planets. I guess I can use player conditions instead.

Reply:
```suggestion
	source
		attributes "near earth" "dirt belt"
```

Reply:
The original filter doesn't include New Wales and Heartland because the filter requires a planet to sell the "Volcano Afterburner" to match, but those two worlds do not sell that afterburner.
They only have "Basic Outfits" and "Ammo South" while the other three worlds have "Common Outfits" which includes the Volcano.
The new outfitter added here doesn't contain the Volcano. Did you want it to add that where it isn't already available?
It does seems a bit odd for these two places that only have very basic outfits to get the bigger afterburner but still not sell the smaller one.

Also, I think you can just list the five planets in the filter:
```
location
	planet "Earth" "Luna" "New China" "New Wales" "Heartland"
```
Tested with `--matches`:
![image](https://github.com/endless-sky/endless-sky/assets/20605679/af6854b8-bc1c-4cc6-813f-b48ba4f89feb)

Reply:
```suggestion
	source
		location
			planet "Earth" "Luna" "New China" "New Wales" "Heartland"
```

Reply:
Yes that was an oversight. Thanks for catching that.

Reply:
```suggestion
	source
		planet "Earth" "Heartland" "Luna" "New China" "New Wales"
```